apps/server/src/services/runtime-service/RuntimeService.ts

```diff
@@ -525,8 +525,8 @@ class RuntimeService {
     }
     const didStop = this.eventTimer?.stop();
     if (didStop) {
-      const newState = state.timer.playback;
-      logger.info(LogOrigin.Playback, `Play Mode ${newState.toUpperCase()}`);
+      const newPlaybackState = runtimeState.getState().timer.playback;
+      logger.info(LogOrigin.Playback, `Play Mode ${newPlaybackState.toUpperCase()}`);
```

Comment:
this was copy-pasted so we need to make the same change on the other functions
also, how about simplifying like 

```suggestion
      const newState = runtimeState.getState();
      logger.info(LogOrigin.Playback, `Play Mode ${newState.timer.playback.toUpperCase()}`);
```

Reply:
think it was the only one that did not get new state after the playback action